profile/README.md

```diff
@@ -1,79 +1,58 @@
-
-Contributors that are currently creating their own forks of models like Mistral are now able to accelerate their work and avoid the costs of forking and retraining their models.  Once proven out, we expect this to attract an even broader set of contributors that have less AI expertise but have creative ideas on how to extend AI models.
+**InstructLab is here to solve these problems.**
 
----
+The project enables community contributors to add additional "skills" or "knowledge" to a particular model ([read more about these in the `taxonomy` README](https://github.com/instructlab/taxonomy/blob/main/README.md)).
 
-## InstructLab is composed of the following projects
+InstructLab's model-agnostic technology gives model upstreams with sufficient infrastructure resources the ability to create regular builds of their open-source licensed models not by rebuilding and retraining the entire model but by composing new skills into it.
 
-### [Taxonomy](https://github.com/instructlab/taxonomy)
+Take a look at "lab-enhanced" models on the [InstructLab Hugging Face](https://huggingface.co/instructlab) page.
 
-The InstructLab method is driven by taxonomies, which are largely created manually and with care. This repository contains a taxonomy tree that will allow you to create models tuned with your data (enhanced via synthetic data generation) using the InstructLab method.
+## Get Started with InstructLab
 
-* [Repo](https://github.com/instructlab/taxonomy)
-* [Contributing](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md)
+* Check out the [Community README](https://github.com/instructlab/community/blob/main/README.md) to get started with using the project. (Contribution guides are included in this document.)  
+* You may wish to read through the [project's FAQ](https://github.com/instructlab/community/blob/main/FAQ.md) to get more familiar with all aspects of InstructLab.
+* You can find all the ways to collaborate with project maintainers and your fellow users of InstructLab beyond GitHub by visiting our [project collaboration](https://github.com/instructlab/community/blob/main/Collaboration.md) page.
 
-### [Command-line Interface](https://github.com/instructlab/instructlab)
+## Code of Conduct
 
-This command-line interface for InstructLab will allow you to create models tuned with data you provide using the InstructLab method on your laptop or workstation.
+Participation in the InstructLab community is governed by our [Code of Conduct](https://github.com/instructlab/community/blob/main/CODE_OF_CONDUCT.md)
 
-* [Repo](https://github.com/instructlab/instructlab)
-* [Contributing](https://github.com/instructlab/instructlab/blob/main/CONTRIBUTING/CONTRIBUTING.md)
+## Quick Links
```

Comment:
Seems there are no links here - should we remove this section?

Reply:
It's a heading with a set of "quick links" type subsections (or that is how I read it anyway)

Reply:
Ohhhh yes that makes sense - if that's the case @lhawthorn can resolve and ignore this comment for me